.gitignore

```diff
@@ -1,8 +1,11 @@
-/build
-/app/build
-/.idea
-/.gradle
 *.iml
-
-# Local configuration file (sdk path, etc)
-local.properties
\ No newline at end of file
+.idea
+.idea/*
+.gradle
+/local.properties
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+google-services.json
+.gradle/
```

Comment:
why not gitignore.io? https://www.toptal.com/developers/gitignore?templates=android,androidstudio,macos